src/App.tsx

```diff
@@ -1,61 +1,94 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
```

Comment:
[CODE STYLE] - Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. Consider moving the data fetching and preparation to a separate service or context provider to keep the App component clean and maintainable.